Sources/ArrayExtensions.swift

```diff
@@ -132,6 +132,13 @@ extension Array where Element: Equatable {
         self.remove(at: index)
     }
 
+    /// EZSE: Removes equal items of array
```

Comment:
Ok, I'm going to do that. Should be a good idea to do your first thought?
 func removeRepeatedElemets()